src/TMedia.h

```diff
@@ -83,15 +83,15 @@ class TMediaData
 private:
     int mMediaProtocol;
     int mMediaType;
-    QString mMediaFileName;
+    QString mMediaFileName {};
```

Comment:
But aren't these initialised to be `null` `QString`s **even *without*** an initialiser here (or in the constructor initialiser list)?

Reply:
I don't know the exact details - could be `null` and that you can't compare `null` one to QString, could be something else, but it fixes the problem.

Reply:
Fundamentally I do not think this - and the changes on line 91 to 94 are making the slightest difference! The default constructor for a `QString` is used automagically and putting a `{}` as an initialiser here (or including the `QString` member in the constructor's list with a default `()` argument) has precisely the same effect as not explicitly initialising them at all...

🤓